library/src/main/java/com/novoda/downloadmanager/lib/BatchStatusUpdater.java

```diff
@@ -0,0 +1,88 @@
+package com.novoda.downloadmanager.lib;
+
+import android.content.ContentResolver;
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.util.SparseIntArray;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.novoda.downloadmanager.lib.Downloads.Impl.*;
+
+class BatchStatusUpdater {
+
+    private static final List<Integer> PRIORITISED_STATUSES = Arrays.asList(
+            // Cancelled
```

Comment:
Removed the ones that are redundant: https://github.com/novoda/download-manager/commit/6f37ca1736f8b5588717fa06ec6f7ca7e790298c